zerver/views/home.py

```diff
@@ -37,20 +40,48 @@ def accounts_accept_terms(request: HttpRequest) -> HttpResponse:
     if request.method == "POST":
         form = ToSForm(request.POST)
         if form.is_valid():
-            assert settings.TERMS_OF_SERVICE_VERSION is not None
+            assert (
+                settings.TERMS_OF_SERVICE_VERSION is not None
+                or request.user.tos_version == UserProfile.TOS_VERSION_BEFORE_FIRST_LOGIN
+            )
```

Comment:
This could be a follow-up, but is this `assert` correct? If the user has two browser windows opened to the app, and clicks continue in one and then the other, I suspect we'll hit this case; should we just delete the assert statement, and maybe only call `do_change_tos_version` if the value is different?